ENIGMAsystem/SHELL/Platforms/General/PFsystem.h

```diff
@@ -36,6 +36,7 @@ enum {
   os_winphone = 7,
   os_tizen = 8,
   os_win8native = 9,
+  os_bsd = 10,
```

Comment:
You're not supposed to have comma on last enum. I think some compilers will cry or throw warnings

Reply:
i didn't know that mainly because that's how it already was before i created this pr, but thanks for the info better to correct that now than later.